web-assembly/hosted/okta-hosted-login/README.md

```diff
@@ -73,7 +73,14 @@ Open the `appsettings.json` file located in the `wwwroot` folder and replace the
 
 #### Server
```

Comment:
It seems we still have instructions to run the webserver, can you please review the readme and make sure the instructions to run the example are correct? 

See this section for example:

_#### Run the web application from Visual Studio
If you run this project in Visual Studio it will start the web application on port 44314 using HTTPS. You can change this configuration in the `launchSettings.json`. Make sure that `okta-blazor-web-assembly-example.Server` is selected as your startup project._